frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss

```diff
@@ -34,13 +34,16 @@
 }
 
 .co-status-card__alert-item {
-  align-items: center;
   display: flex;
   padding-bottom: 0.5rem;
   padding-left: var(--pf-c-card--child--PaddingLeft);
   padding-right: var(--pf-c-card--child--PaddingRight);
 }
 
+.co-status-card__alert-item-icon {
+  display: flex;
```

Comment:
Do you need this? Without it:
<img width="929" alt="Screen Shot 2019-11-12 at 12 33 56 PM" src="https://user-images.githubusercontent.com/287880/68690635-b8892000-0548-11ea-8819-8033f2714bcc.png">

Reply:
See [this comment](https://github.com/openshift/console/pull/3319#issuecomment-552492065) for some history. There's some invisible margin around the SVG with its default inline display mode that prevents us from aligning it quite nicely enough with the date. Setting `flex` here isn't perfect either, but it's the easiest way I'm seeing of getting close enough without restructuring the HTML to more correctly lay out the icon/date/action within the same element.

Compare this image without flex to the image above. Slightly less good, but 🤷‍♂️:

![image](https://user-images.githubusercontent.com/9122899/68692189-0a2eac00-0543-11ea-8cf1-b2d88b23ad07.png)

Reply:
ah, ya without flex it seems to align nicely to the top which looked ok to me, but I guess the standard is centre aligned.